maventa/maventa.ts

```diff
@@ -0,0 +1,95 @@
+import https from 'https';
+import { getAccessToken } from './auth';
+import config from './config';
+import Invoice from './types/invoice';
+
+// Send company invoice dependencies
+
+// import fs from 'fs';
+// import FormData from 'form-data';
+
+export async function listInvoices(): Promise<Invoice[]> {
+  return new Promise((resolve, reject) => {
+    try {
+      getAccessToken().then(async (token) => {
+        const options = {
+          hostname: config.baseUrl,
+          path: `/v1/invoices?`,
+          method: 'GET',
+          headers: {
+            'Authorization': `Bearer ${token}`,
+            'Accept': 'application/json',
+            'User-Api-Key': config.clientSecret,
+            'Company-UUID': config.clientId,
+          },
+        };
+
+        const responseData: string[] = [];
+        const req = https.request(options, (res) => {
+          res.on('data', (chunk) => {
+            responseData.push(chunk);
+          });
+          res.on('end', () => {
+            const invoices = JSON.parse(responseData.join(''));
+            resolve(invoices);
+          });
+        });
+
+        req.on('error', (e) => {
+          reject(e);
+        });
+
+        req.end();
+      });
+    } catch (error) {
+      console.error('Error listing invoices:', error);
+      reject(error);
+    }
+  });
+}
+
+// Sends invoice to configs CompanyEDI
+
```

Comment:
It's bad style to left commented code in the code base. Leave it as uncommented or remove.